image-definitions/kubeflow/components/crud-web-apps/jupyter/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# --- Build the backend kubeflow-wheel ---
```

Comment:
In this file versions builder versions were bumped up to `bullseye`. This will be downgraded.